actionview/lib/action_view/helpers/cache_helper.rb

```diff
@@ -184,7 +184,7 @@ def cache_fragment_name(name = {}, options = nil)
         if skip_digest
           name
         else
-          fragment_name_with_digest(name)
+          fragment_name_with_digest(name, virtual_path: options.try(:[], :virtual_path))
```

Comment:
If we're changing a method anyway, it's okay to switch it to use keyword arguments completely.